components/tools/OmeroWeb/test/integration/test_tree.py

```diff
@@ -1782,9 +1782,9 @@ def test_marshal_images_thumb_version(self, userA, image_pixels_userA):
         Test marshalling image, loading thumbnail version
         """
         conn = get_connection(userA)
-        # Thumbnail not yet created: version will be -1
+        # Thumbnail is created, see testlib.ITest.createTestImage
         expected = expected_images(userA, [image_pixels_userA],
-                                   extraValues={'thumbVersion': -1})
+                                   extraValues={'thumbVersion': 0})
```

Comment:
If the behaviour of ```testlib.ITest.createTestImage``` has changed then we need to use another way of creating images without a thumbnail to test that the initial version is -1 (e.g. during pyramid generation).